dbtvault-dev/macros/supporting/hash.sql

```diff
@@ -26,7 +26,8 @@
     {%- set hash_size = 16 -%}
 {%- endif -%}
 
-{%- set standardise = "NULLIF(UPPER(TRIM(CAST([EXPRESSION] AS VARCHAR))), '')" %}
+{%- set hash_case = var('hash_case', 'UNCHANGED') -%}
```

Comment:
Should be `UPPER` to ensure normal working from previous versions of dbtvault for people who do not need to configure this